source/blog/2016-05-24-atomic-app-0-5-2-release.html.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Atomic App 0.5.2 released: Cleaner logging and binary generation!
+author: cdrage
+date: 2016-05-24 12:15:00 UTC
+tags: atomicapp, Nulecule, releases
+published: true
+comments: true
+---
+
+In this weeks release of Atomic App we not only have cleaner logging output when using the tool, but we've also added binary generation!
+
+The main features of this release are:
+  - Cleaning logging
+  - Binary generation
+  - README.MD generation removed from `atomicapp init`
+
+READMORE
+
+Other:
+
+  - Code cleanup / minor code modifications (dictionary searches, loops)
+  - Provider module / class loading refactoring
+
+## Cleaner logging
+
+We've completed a clean-up on how we log Atomic App. With this release we've added ASCII art as well as cleaned up a lot of the redundant information outputted when deploying an Atomic App.
+
+```
+▶ atomicapp run projectatomic/helloapache --destination helloapache --provider=docker
+INFO   ::    _  _             _      _
+INFO   ::   /_\| |_ ___ _ __ (_)__  /_\  _ __ _ __   Version:  0.5.1
+INFO   ::  / _ \  _/ _ \ '  \| / _|/ _ \| '_ \ '_ \  Nulecule: 0.0.2
+INFO   :: /_/ \_\__\___/_|_|_|_\__/_/ \_\ .__/ .__/  Mode:     Run
+INFO   ::                                |_|  |_|
+INFO   :: Unpacking image: projectatomic/helloapache to helloapache
+INFO   :: Skipping pulling Docker image: projectatomic/helloapache
+INFO   :: Extracting nulecule data from image: projectatomic/helloapache to helloapache
+INFO   :: Deploying to provider: Docker
+
+Your application resides in helloapache
+Please use this directory for managing your application
+```
+
+## Binary generation
+
+You'll now be able to generate a stand-alone binary for Atomic App without the need of having Python installed onto your system. This increases portability and is compatible with all Linux-based operating systems.
```

Comment:
"Increases portability" - let's not make that statement. One could actually argue it decreases portability to other CPU architectures because it's now a binary.

I say let's just make this one sentence: `You'll now be able to generate a stand-alone binary for Atomic App without the need of having Python installed`.